ajax/libs/simple-slider/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "simple-slider",
+  "version": "1.0.0",
+  "description": "The 1kb JavaScript Carousel",
+  "npmName": "simple-slider",
```

Comment:
There are 19 releases on GitHub but 16 on NPM. I think we should set git auto-update config. @ruyadorno How do you think?